Uplift/Helpers/OnboardingAnimations.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  OnboardingAnimations.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/26/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Presentation
+import UIKit
+
+// MARK: - Arrow Button Transition
+
+/// Transition that changes arrow colors when swiping
+public class ArrowButtonTransition: NSObject, Animatable {
+    private let content: Content
+    private let duration: TimeInterval
+    private let delay: TimeInterval
+    private var played = false
+
+    var transitionIsEnabled = false
+
+    public init(content: Content,
+                duration: TimeInterval = 1.0,
+                delay: TimeInterval = 0.0) {
+        self.content = content
+        self.duration = duration
+        self.delay = delay
+
+        super.init()
+    }
+
+    private func animate() {
+        UIView.animate(
+            withDuration: duration,
+            delay: delay,
+            options: [.beginFromCurrentState, .allowUserInteraction],
+            animations: ({ [unowned self] in
+                if let button = self.content.view as? UIButton {
+                    button.backgroundColor = self.transitionIsEnabled ? .primaryYellow : .none
+                    button.isEnabled = self.transitionIsEnabled
+                }
+            }),
+            completion: nil
+        )
+
+        played = true
+    }
+}
```

Comment:
newline before here and same with the other `extension` and `class` in this file